sdk/schema/fieldname/names.go

```diff
@@ -5,6 +5,7 @@ import "github.com/1Password/shell-plugins/sdk"
 // Credential field names.
 const (
 	APIHost         = sdk.FieldName("API Host")
+	API_URL         = sdk.FieldName("API URL")
```

Comment:
nit: We didn't use the underline _ convention so far in naming these, so let's rephrase this to `APIUrl` (even though URL is all caps normally)

Reply:
Thanks, Updated this change